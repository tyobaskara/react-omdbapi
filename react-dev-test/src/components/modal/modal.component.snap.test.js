import React from 'react';
import renderer from 'react-test-renderer';
import Modal from './modal.component';

test('Modal snaptest', () => {
  const props = {
    children: 'children'
  };
  const component = renderer.create(
    <Modal {...props} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});