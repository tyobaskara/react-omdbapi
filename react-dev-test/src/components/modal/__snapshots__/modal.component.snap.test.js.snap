// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal snaptest 1`] = `
<div
  className="modal"
>
  <div
    className="overlay"
  />
  <div
    className="modal-container"
  >
    <button
      className="modal-close"
    >
      <i
        className="fa fa-close"
      />
    </button>
    children
  </div>
</div>
`;
